src/shared/format-table.c

```diff
@@ -1377,7 +1377,7 @@ static const char *table_data_format(Table *t, TableData *d, bool avoid_uppercas
 
         case TABLE_BPS: {
                 _cleanup_free_ char *p;
-                size_t n;
+                uint64_t n;
```

Comment:
`size_t` is appropriate here, because it is used for memory size. So please drop this change.

Reply:
It isn't – it's used for the speed in bits per second.

See the single use of it, which is in networkctl:
https://github.com/systemd/systemd/blob/a6e1018df27d89b21ea254236692b5af38402776/src/network/networkctl.c#L1795-L1802

`info->speed` is
https://github.com/systemd/systemd/blob/a6e1018df27d89b21ea254236692b5af38402776/src/network/networkctl.c#L182-L186

Using `size_t` here is (a) plain wrong, since the vararg is `uint64_t`, and (b) admittedly saved by little endian for values smaller than 2^32, but primarily (c) *breaks the output* when the speed is more than 2^32, like any interface faster than 4G, which are most of this table: https://github.com/systemd/systemd/blob/a6e1018df27d89b21ea254236692b5af38402776/src/shared/ethtool-util.c#L62

Reply:
This is a local variable, and it's used in lines 1389-1390 below for `strscpy`. It's not used anywhere else!

Reply:
Oh wait, you're right; I misread this entirely, sorry!